exec-tests

```diff
@@ -243,7 +243,12 @@ if [[ "${major}" == "all" || "${major}" == "server" ]]; then
         shift
         posargs="${@}"
         # We use SQLALCHEMY_SILENCE_UBER_WARNING here ... (see above).
-        REQUESTS_CA_BUNDLE=${PWD}/server/pbenchinacan/etc/pki/tls/certs/pbench_CA.crt SQLALCHEMY_SILENCE_UBER_WARNING=1 PYTHONUNBUFFERED=True PBENCH_SERVER=${server_arg} KEEP_DATASETS="${keep_datasets}" pytest --tb=native -v -s -rs --pyargs ${posargs} pbench.test.functional.server
+        SQLALCHEMY_SILENCE_UBER_WARNING=1 \
```

Comment:
I think this change is wrong:

> requests.exceptions.SSLError: HTTPSConnectionPool(host='localhost', port=8443): Max retries exceeded with url: /api/v1/endpoints (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))

... although I don't mind the reformatting.

Reply:
Um, yeah.  That was the result of an impressive brain-cramp.  (Thanks for pointing that out!...although, with the failing tests, I assume I would have tumbled to the issue as well.  🥴)

I've updated the PR with the intended change and, based on your preference, included this change (corrected) as well.